styleguide/components/31-molecules/teasers/teaser-topic/teaser-topic.twig

```diff
@@ -7,6 +7,18 @@
         'text': teaser_paragraph_text
       } %}
 
+      <ul class="links teaser-topic-links">
+        {% for link in teaser_topic_links %}
+          <li>
+            {% include '@link' with {
+              'text': link.text,
+              'aria_labelledby': link.uid,
```

Comment:
Why are these aria-labelledby?
Link text should be sufficient, with the exception of read-more links*

*regarding read-more links, we can no longer use aria-labelledby as it's not conform WCAG 2.1 2.5.3 
"For user interface components with labels that include text or images of text, the name contains the text that is presented visually."

We'd better replace them by hidden spans, is this feasible in this ticket or would you prefer a bugticket?

Reply:
Good point. Better to create a bug ticket for this 🙂